### ISSUE TITLE
WIDGET: Not possible to purchase same item more than once within one transaction

### ISSUE THREAD

**Issue by wszarmach** [krico/jas#624]
Currently the widget only allows the user to select each activity or activity package once within a transaction.
Already we have had users who wanted to purchase the same activity more than once within the same transaction.
The benefits of making this change are
* Reduced commission linked payments, eg PayPal
* Reduced repetitive work for a user, does not have to perform the same checkout process twice (perhaps making a mistake)

**Comment by pauljanes** [comment]
Yes the only thing is that Jayne wanted to know guest names. So there would need to be possibility to enter guest names (optional). By the way we need a name field not just email. Some people use weird emaila but provider wants to know who is coming to class. We need to look at user creation as we need additional optional fields like contact number, address etc

**Comment by pauljanes** [comment]
I also wanted to mention i read a useability study that said dont assume people in checkout will know that setting items to 0 is same as deleting. It said you should allow both - setting item count to 0 then updating basket or delete button like we have now
